gateway/api_test.go

```diff
@@ -2745,12 +2794,12 @@ func testPatchOAS(t *testing.T, ts *Test, api oas.OAS, params map[string]string,
 	ts.Gw.DoReload()
 }
 
-func testImportOAS(t *testing.T, ts *Test, queryparams string, testCase test.TestCase) {
-	pathPath := "/tyk/import/oas"
-	if queryparams != "" {
-		pathPath += "?" + queryparams
+func testImportOAS(t *testing.T, ts *Test, queryParams string, testCase test.TestCase) {
+	importPath := "/tyk/import/oas"
+	if queryParams != "" {
```

Comment:
we've a `QueryParams` in `testCase` as map[string]string, we could use that for easiness instead of string. Setting query params in test case would handle the URL encoding as well